communitter-frontend/src/components/NewInvitationModal.js

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from "react";
+import { Button, Form, Modal } from "react-bootstrap";
+import { useSelector } from "react-redux";
+import { getUserRoleValue, roles } from "../utilities/roles";
+
+const NewInvitationModal = ({ show, onHide, onSubmit, isLoading }) => {
```

Comment:
Why do you named it as NewInvitationModal ?

Reply:
as it serves to send new invitation and it is a modal.